compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/AbstractParserErrorHandler.java

```diff
@@ -478,7 +478,7 @@ public Result(ArrayDeque<Solution> fixes, int matches) {
      *
      * @since 1.2.0
      */
-    enum Action {
+    public enum Action {
```

Comment:
This doesn't looks like a cause for java doc issue. Do you have the log for this?

Reply:
Yes, The `./gradlew :ballerina-parser:createJavadoc` command produces the following warnings 

- `/path/to/repo/ballerina-lang/compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/AbstractParserErrorHandler.java:468: warning - Tag @link: reference not found: Action#KEEP`

- `/path/to/repo/ballerina-lang/compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/AbstractParserErrorHandler.java:424: warning - Tag @link: reference not found: Action`

and 
`> Task :ballerina-parser:createJavadoc FAILED`
`FAILURE: Build failed with an exception.`